packages/validators-util/src/isChassisNumber/isChassisNumber.test.ts

```diff
@@ -0,0 +1,14 @@
+import { cleanup } from "@testing-library/react";
+import { isChassisNumber } from "./isChassisNumber";
+
+describe("isChassisNumber", () => {
+    afterAll(cleanup);
+
+    it("expect to be CORRECT chassis number", () => {
```

Comment:
Nå som det faktisk kommer flere ting inn her, så syns jeg vi skal skrive litt mer utfyllende tester. Dette legger grunnlaget for denne pakka, som om vi vil ha decent testdekning her, så bør vi nok starte med litt mer enn dette. Med mindre noen lover på tro og ære at de skal fikse det senere(og faktisk gjør det).

Reply:
Enig med dette! Nå er fortsatt de fleste reglene her ganske basic, men er greit å få testet edge cases ordentlig. Synes uansett vi bør formulere navnet på testene bedre. "expect to be correct chassis number" sier ingenting om _hvorfor_ testen skal returnere det den gjør. 

Anbefaler egentlig å skrive alle tingene som skal være oppfylt som en liste, og så bare skrive dem som tester. Det er vel derfor funksjonene heter det de gjør her også ;)

```suggestion
    it("should return true for 9-digit numbers", () => {
```

Reply:
Er ikke bare for testenes skyld heller, tester er ofte den beste måten å finne ut hva som var den eeegentlige intensjonen til koden. Så vi bør ha et sett med positive og negative tester.